cmd/kind/get/clusters/clusters.go

```diff
@@ -30,9 +30,10 @@ func NewCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Args: cobra.NoArgs,
 		// TODO(bentheelder): more detailed usage
-		Use:   "clusters",
-		Short: "lists existing kind clusters by their name",
-		Long:  "lists existing kind clusters by their name",
+		Use:     "clusters",
+		Aliases: []string{"cluster"},
```

Comment:
i'm not a big fan of very similar aliases, but maybe others will be OK with that.